make-release.sh

```diff
@@ -29,11 +29,11 @@ Options:
 "
 }
 
+set -e
+
 if [[ ! ${VERSION} ]]; then
   usage
   exit 1
-else # clone into a temp folder so we don't collide with local changes to this script
```

Comment:
I added this so that it would be easier to test local changes to this script w/o a checkout failing because of dirty workspace. But I guess for a GH action it's a redundant checkout as the code's already present from an earlier action step.

Reply:
maybe it's better to implement this as an option, so we can run locally in /tmp if we want, but otherwise just use current folder's contents

Reply:
sure